workshop/exercise-1/README.md

```diff
@@ -6,92 +6,97 @@ You must already have a [cluster created](https://console.bluemix.net/docs/conta
-    ibmcloud login --apikey {apikey}
-    ```
-    
+2.  Log in to the IBM Cloud CLI. If you have a federated account, include the `[--sso]` flag.
+
+```shell
+ibmcloud login [--sso]
+```
+
+or if you have an api key, use:
+
+```shell
+ibmcloud login --apikey {apikey}
+```
+
 3.  Install the IBM Cloud Kubernetes Service plug-in.
-    ```bash
-    ibmcloud plugin install container-service -r Bluemix
-    ```
+
+```shell
+ibmcloud plugin install container-service -r Bluemix
+```
 
 4. To verify that the plug-in is installed properly, run `ibmcloud plugin list`. The Container Service plug-in is displayed in the results as `container-service`.
 
-5.  Initialize the Container Service plug-in and point the endpoint to your region. For example when prompted, enter `5` for `us-east`.   
-    ```bash
-    $ ibmcloud cs region-set
-    Choose a region:
-    1. ap-north
-    2. ap-south
-    3. eu-central
-    4. uk-south
-    5. us-east
-    6. us-south
-    Enter a number> 5
-    ```
-    
-6. Install the Kubernetes CLI. Go to the [Kubernetes page](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-binary-via-curl) to install the CLI and follow the steps.
+5.  Initialize the Container Service plug-in and point the endpoint to your region. For example when prompted, enter `5` for `us-east`.
 
+Example:
+
+```shell
+$ ibmcloud cs region-set
+Choose a region:
+1. ap-north
+2. ap-south
+3. eu-central
+4. uk-south
+5. us-east
+6. us-south
+Enter a number> 5
+```
+
+6. Install the Kubernetes CLI. Go to the [Kubernetes page](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-binary-via-curl) to install the CLI and follow the steps.
 
 ## Access your cluster
 Learn how to set the context to work with your cluster by using the `kubectl` CLI, access the Kubernetes dashboard, and gather basic information about your cluster.
 
 1.  Set the context for your cluster in your CLI. Every time you log in to the IBM Cloud Kubernetes Service CLI to work with the cluster, you must run these commands to set the path to the cluster's configuration file as a session variable. The Kubernetes CLI uses this variable to find a local configuration file and certificates that are necessary to connect with the cluster in IBM Cloud.
 
-    a. List the available clusters.
-    
-    ```bash
-    ibmcloud cs clusters
-    ```
-    
-    b. Download the configuration file and certificates for your cluster using the `cluster-config` command.
-    
-    ```bash
-    ibmcloud cs cluster-config <your_cluster_name>
-    ```
-    
-    c. Copy and paste the output command from the previous step to set the `KUBECONFIG` environment variable and configure your CLI to run `kubectl` commands against your cluster.
+a. List the available clusters.
+
+```shell
+ibmcloud cs clusters
+```
+
+b. Download the configuration file and certificates for your cluster using the `cluster-config` command.
+
+```shell
+ibmcloud cs cluster-config <your_cluster_name>
```

Comment:
We may want to be consistent... here we're using <...> for vars but way up top we use {...} (e.g. apikey)

Reply:
I know this isn't your fault, but I just noticed it  :-)

Reply:
converted apikey to `<>`